libs/chlorophyll/scss/components/select/_mixins.scss

```diff
@@ -7,12 +7,15 @@
   position: relative;
 
   select {
+    @include mixins.padding-horizontal(5);
```

Comment:
Not sure the padding was the same as what it comes now, but it looks good in storybook, also this is the same setting that input has.

Reply:
The select needs additional right-hand padding to allow more space for the chevron, but otherwise it is correct. I added `padding-right` now, so you don't need to do anything.